<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QMainWindow" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="CB_CACHE">
      <property name="text">
       <string>Use code cache (bruteforce bypass)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="CB_KILLGFCLIENT">
      <property name="text">
       <string>Kill gfclient.exe on startup</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>PlatformGameId</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_PLATFORMGAMEID">
        <property name="inputMask">
         <string>xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="PB_BROWSE">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="CB_CHECK_GITHUB">
      <property name="text">
       <string>Check for launcher update on startup</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Time between each connection (s)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="LE_NTDIR"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="SB_TIME">
      <property name="maximum">
       <double>3600.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>12.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="PB_REPAIR">
        <property name="text">
         <string>Repair game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_UPDATE">
        <property name="text">
         <string>Update game</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
